apps/site/assets/ts/stop/components/DisplayTime.tsx

```diff
@@ -94,49 +77,34 @@ function BaseTimeDetails(): JSX.Element {
 }
 
 function DelayedTimeDetails(): JSX.Element {
-  const { departure, isLessThanHourAway, isCR, targetDate } = useContext(
-    DepartureContext
-  );
-  const { prediction, schedule } = departure;
-  const strikedthroughSchedule = (
-    <BasicTime
-      displayType="absolute"
-      time={schedule!.time}
-      targetDate={targetDate}
-      strikethrough
-    />
-  );
-  if (isLessThanHourAway && isCR) return strikedthroughSchedule;
-  if (!isLessThanHourAway) return strikedthroughSchedule;
+  const { departure, targetDate } = useContext(DepartureContext);
+  const scheduledTime = departure.schedule!.time;
   return (
     <>
-      {"Delayed "}
+      Delayed{" "}
       <BasicTime
         displayType="absolute"
-        time={prediction!.time}
+        time={scheduledTime}
         targetDate={targetDate}
-      />{" "}
-      {strikedthroughSchedule}
+        strikethrough
+      />
     </>
   );
 }
 
 const CancelledTimeDetails = (): JSX.Element => {
-  const { time, targetDate } = useContext(DepartureContext);
-
+  const { departure, targetDate } = useContext(DepartureContext);
+  const scheduledTime = departure.schedule!.time;
```

Comment:
Is schedule guaranteed to be defined here?  I know we have a help to grab times from the `departure` object

Reply:
It _should_ under the `isCancelled` condition, but that's indirect, so I've added an additional check for the schedule.